### PR TITLE
Fix Pike state delta subscription

### PR DESCRIPTION
**Pull request by RyanLassigBanks**
Fixes event filter for Pike state delta events

Signed-off-by: Ryan Banks <rbanks@bitwise.io>

### CODE REVIEW

File: daemon/src/event/block.rs [hunk 1]
```diff
@@ -57,6 +57,8 @@ impl EventHandler for BlockEventHandler {
             block.block_id, block.block_num, block.state_root_hash
         );
 
+        debug!("The following operations will be performed: {:#?}", db_ops);
```
Comment (agunde406) [inline daemon/src/event/block.rs]: This is removed in the next commit. If it should be removed it should be removed from this commit.

Comment (agunde406) [reply daemon/src/event/block.rs]: Same with derive debug below

Comment (peterschwarz) [inline daemon/src/event/block.rs]: After @agunde406  pointed this message out, I'm not sure this is a good debug message.  Just trying to think about debugging this from a sysadmin perspective - it's potentially a huge message.

Comment (RyanLassigBanks) [reply daemon/src/event/block.rs]: I see your point. I would argue that debug level messages are suppose to over communicate, but I could also see it potentially being cumbersome under some circumstances. I think some logging there would be useful. What would you suggest?

Comment (RyanLassigBanks) [reply daemon/src/event/block.rs]: `debug!` -> `trace!`

**Pull request merged**